arm/Microsoft.Authorization/roleDefinitions/deploy.bicep

```diff
@@ -74,6 +77,7 @@ module roleDefinition_rg '.bicep/nested_roleDefinitions_rg.bicep' = if (empty(ma
     assignableScopes: !empty(assignableScopes) ? assignableScopes : []
     subscriptionId: subscriptionId
     resourceGroupName: resourceGroupName
+    cuaId: !empty(cuaId) ? cuaId : ''
```

Comment:
Please remove any 'passing' of the cuaId. We only apply it to a template directly.

Reply:
Done